### PR TITLE
<Description> : Adding is in future and is in past extensions.

### PR DESCRIPTION
**Pull request by Khalian**
## Checklist

- [x] New Extension
- [x] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/DateExtensions.swift [hunk 1]
```diff
@@ -111,6 +111,16 @@ extension Date {
             return "\(components.second) seconds ago"
         }
     }
+    
+    /// EZSE: Check if date is in future.
+    public var isInFuture: Bool {
```
Comment (piv199) [inline Sources/DateExtensions.swift]: isFuture

File: Sources/DateExtensions.swift [hunk 2]
```diff
@@ -111,6 +111,16 @@ extension Date {
...
+        return self > Date()
+    }
+    
+    /// EZSE: Check if date is in past.
+    public var isInPast: Bool {
```
Comment (piv199) [inline Sources/DateExtensions.swift]: isPast

File: Sources/DateExtensions.swift (hunk 1 above)
Comment (Khalian) [inline Sources/DateExtensions.swift]: Done

**Comment by goktugyil** [comment]
In what cases would we need this?

**Comment by Khalian** [comment]
Off the top of my head running cron jobs or a trigger after a particular date.

**Comment by piv199** [comment]
Could you also add to changelog, but maybe not in this pr

**Comment by Khalian** [comment]
We can punt this to the next release and tag the last release as official.

**Pull request merged**